lib/id3c/cli/command/etl/manifest.py

```diff
@@ -208,7 +208,7 @@ def upsert_sample(db: DatabaseSession,
             update warehouse.sample
                set identifier = %(identifier)s,
                    collection_identifier = %(collection_identifier)s,
-                   collected = date_or_null(%(collection_date)s),
+                   collected = coalesce(date_or_null(%(collection_date)s), collected),
```

Comment:
I'm doing exactly the same thing in PR https://github.com/seattleflu/id3c/pull/160 because the incident report records won't give us collected date.